### ISSUE TITLE
Training object detection model

### ISSUE THREAD

**Issue by ivbelkin** [irfanICMLL/structure_knowledge_distillation#5]
Great work!

I see, you present results on COCO object detection, but I cant find training procedure and which loss function did you use. Could you share it please?

Thank you!

**Comment by irfanICMLL** [comment]
I just use the code of FCOS: https://github.com/tianzhi0549/FCOS , and you can modify their code by adding the distillation loss we provided. I use the output logits and the features from all the towers. If you have any problems, please feel free to contact me~
